package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "@heroicons/react": "^2.1.5",
     "@hookform/resolvers": "^3.9.0",
+    "@lucia-auth/adapter-drizzle": "^1.0.7",
```

Comment:
why did you install this dependency? We are not using drizz;e

Reply:
i installed it bc of a lucia tutorial. thought it would be standard for using it 🥸